### PR TITLE
Documentation on how to enable Logging

### PR DESCRIPTION
**Pull request by khrm**
This document how to install results for logging.

Addresses  #407

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you review them:

- [x] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] [Tested your changes locally](https://github.com/tektoncd/results/blob/main/docs/DEVELOPMENT.md) (if this is a code change)
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including functionality, content, code)
- [x] Has a kind label. You can add a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user-facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [ ] Release notes contain the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by adambkaplan** [comment]
@khrm we already have an installation doc - does it make more sense to add a section instead of creating a new markdown doc?

https://github.com/tektoncd/results/blob/main/docs/install.md

File: Logging_Enabled_installation.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+          name: tekton-results-api-config
+        name: config
+      - name: tls
+        secret:
+          defaultMode: 420
+          secretName: tekton-results-tls
+      - name: tekton-logs
+        persistentVolumeClaim:
+            claimName: tekton-logs
+EOF
```
Comment (avinal) [inline Logging_Enabled_installation.md:150]: Would it be better if we put a kustomize patch (or kubernetes patch) instead of the full YAML.

**Comment by sayan-biswas** [comment]
@khrm This PR, #447 enables log with kustomize overlays.

**Comment by khrm** [comment]
Closing this then.

**Pull request closed without merging**